### PR TITLE
Feature/multi remote support

### PR DESCRIPTION
**Pull request by monavari-lebrecht**
add multiremote support, have a closer look at examples/ui5-js-app/webapp/test/e2e/multiremote.test.js and examples/ui5-js-app/wdio-multiremote.conf.js for information how to use it.

Basic usage is:
Change capabilities to 
```
capabilities: {
        one: {
            capabilities: {
                browserName: "chrome",
                acceptInsecureCerts: true
            }
        },
        two: {
            capabilities: {
                browserName: "chrome",
                acceptInsecureCerts: true
            }
        }
    },
```

So you can access the browser instances either by calling them by their name to get the control instance like in single remote tests:
```
const button = await browser.two.asControl({
            selector: {
                id: "openDialogButton",
                viewName: "test.Sample.view.Main"
            }
        })
```
or access them in all instances at the same time:
```
const button = await browser.asControl({
            selector: {
                id: "openDialogButton",
                viewName: "test.Sample.view.Main"
            }
        })
```
to get an array of controls from each instance one:
```
        const buttonOne = button[0]
        const buttonTwo = button[1]
```

TODO: writing more tests for each command. At the moment only basic tests are integrated.

### CODE REVIEW

File: client-side-js/getSelectorForElement.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 async function clientSide_getSelectorForElement(oOptions) {
-    return await browser.executeAsync((oOptions, done) => {
+    return await browserInstance.executeAsync((controlSelector, done) => {
```
Comment (dominikfeininger) [inline client-side-js/getSelectorForElement.js:2]: Hi, something misssing here, or just a mixup of changes?

File: client-side-js/allControls.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-async function clientSide_allControls(controlSelector) {
+async function clientSide_allControls(controlSelector, browserInstance) {
     controlSelector = await Promise.resolve(controlSelector) // to plug into fluent async api
-    return await browser.executeAsync((controlSelector, done) => {
```
Comment (dominikfeininger) [inline client-side-js/allControls.js:3]: Why no use of `browserInstance` in files `client-side-js/` `_navTo, getUI5Version, injectTools, injectUI5, testLibrary`?

**Comment by dominikfeininger** [comment]
hi, when running on my machine I get a different output:

image

It passes but throws errors?

**Comment by vobu** [comment]
hey @monavari-lebrecht, can you please resolve the merge conflicts so we can get this cool feature in? thanks!

**Comment by vobu** [comment]
> It passes but throws errors?

FTR and as investigated: the error message is intended as part of the test checking for a non-existent control

**Review by vobu — COMMENTED** [review]
just a minor hunch that there's a duplicate def in there (see respective comment) - otherwise good to go!

**Pull request merged**